subprojects/cfa/cfa/src/main/antlr/CfaDsl.g4

```diff
@@ -227,7 +236,7 @@ bvExtendExpr
 
 unaryExpr
 	:	bitwiseNotExpr
-	|	oper=(PLUS | MINUS | BV_POS | BV_NEG) op=unaryExpr
+	|	oper=(PLUS | MINUS | BV_POS | BV_NEG | FP_ABS | FP_IS_NAN | FPROUNDTOINT | FPSQRT) op=unaryExpr
```

Comment:
There is no way to do FpToFp conversion.

Reply:
Fixed in [2608512](https://github.com/ftsrg/theta/pull/123/commits/260851260a768790fd16a6f98dcd612f29bb0084)